src/main/java/seedu/address/logic/parser/ToDoListParser.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import seedu.address.logic.commands.AddToDoCommand;
+import seedu.address.logic.commands.CommandToDo;
+import seedu.address.logic.commands.DeleteToDoCommand;
+import seedu.address.logic.commands.HelpToDoCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * ToDoList parses user input.
+ */
+public class ToDoListParser {
```

Comment:
we should have only one parser since we only have one command line so I don't think this is required

Reply:
Agree with Bryan. Same with ParserToDo class.

Maybe we can add an attribute to the Command abstract class to indicate whether it is a TodoList command or a Scheduler command. Then in the LogicManager, once we parse the command, we can call the execute method with the appropriate model.